pkgs/unified_analytics/lib/src/analytics.dart

```diff
@@ -418,7 +417,11 @@ class AnalyticsImpl implements Analytics {
     // each event that is sent to Google Analytics -- it will be responsible
     // for getting the session id or rolling the session if the duration
     // exceeds [kSessionDurationMinutes]
-    _sessionHandler = Session(homeDirectory: homeDirectory, fs: fs);
+    _sessionHandler = Session(
+      homeDirectory: homeDirectory,
+      fs: fs,
+      analyticsInstance: this,
```

Comment:
Instead of this circular reference situation, can we just tear-off `Analytics.send` method and pass that function to `Session` and `LogHandler`?

Reply:
Great idea!